sdk/src/main/scala/com/horizen/storage/SidechainStateStorage.scala

```diff
@@ -183,6 +185,25 @@ class SidechainStateStorage(storage: Storage, sidechainBoxesCompanion: Sidechain
     }
   }
 
+  def getForgerListIndexes: Option[ForgerList] = {
```

Comment:
Consider reanming it to getForgerList, we are returning the list (not the indexes)

Reply:
Done